### PR TITLE
srepr docs modified

### PR DESCRIPTION
**Pull request by Shekharrajak**
Resolved the conflict for My PR [#10215](https://github.com/sympy/sympy/pull/10215) for [#10185](https://github.com/sympy/sympy/issues/10185)

### CODE REVIEW

File: doc/src/modules/printing.rst [hunk 1]
```diff
@@ -307,31 +307,53 @@ This class implements Python printing. Usage::
     x = Symbol('x')
     e = 5*x**3 + sin(x)
 
-srepr
------
+Srepr
+-----------
```
Comment (moorepants) [inline doc/src/modules/printing.rst]: srepr isn't capitalized and there are too many underlines.

File: doc/src/modules/printing.rst [hunk 2]
```diff
@@ -307,31 +307,53 @@ This class implements Python printing. Usage::
...
 
 .. module:: sympy.printing.repr
 
-This printer generates executable code. This code satisfies the identity
-``eval(srepr(expr)) == expr``.
```
Comment (moorepants) [inline doc/src/modules/printing.rst]: This note is important to keep in here.

File: doc/src/modules/printing.rst [hunk 3]
```diff
@@ -307,31 +307,53 @@ This class implements Python printing. Usage::
...
+SymPy doesn’t use repr() for generating textual representation of expressions.
 
-``srepr()`` gives more low level textual output than ``repr()``
+Usage::
 
-Example::
+    >>> repr(5*x**3 + sin(x))
+    '       5*x**3 + sin(x)'
```
Comment (moorepants) [inline doc/src/modules/printing.rst]: What's with the leading whitespace?

File: doc/src/modules/printing.rst (hunk 2 above)
Comment (moorepants) [inline doc/src/modules/printing.rst]: Oh, I see it below, nevermind.

File: doc/src/modules/printing.rst [hunk 4]
```diff
@@ -307,31 +307,53 @@ This class implements Python printing. Usage::
...
-  >>> srepr(5*x**3 + sin(x))
-  "Add(Mul(Integer(5), Pow(Symbol('x'), Integer(3))), sin(Symbol('x')))"
+Usage::
 
-``srepr()`` gives the ``repr`` form, which is what ``repr()`` would normally give
-but for SymPy we don’t actually use ``srepr()`` for ``__repr__`` because it’s 
-is so verbose, it is unlikely that anyone would want it called by default.
-Another reason is that lists call repr on their elements, like ``print([a, b, c])``
-calls ``repr(a)``, ``repr(b)``, ``repr(c)``. So if we used srepr for `` __repr__`` any list with 
-SymPy objects would include the srepr form, even if we used ``str()`` or ``print()``.
```
Comment (moorepants) [inline doc/src/modules/printing.rst]: This explanation seems useful too. Why remove it?

**Comment by moorepants** [comment]
This could also be a nice place to show how you can write the results of `srepr` to file to serialize a SymPy expression.

**Comment by Shekharrajak** [comment]
@moorepants  Actually this is the almost same [PR](https://github.com/sympy/sympy/pull/10215).Previously there were no examples and explanation. According to [comment](https://github.com/sympy/sympy/pull/10215#issuecomment-164878230) someone sent the PR with same examples and explanation,got merged too.That made my  [PR](https://github.com/sympy/sympy/pull/10215) conflicted.I have reverted my branch to come out from that conflict.

File: doc/src/modules/printing.rst (hunk 4 above)
Comment (Shekharrajak) [inline doc/src/modules/printing.rst]: These lines are in different place like [this](http://docs.sympy.org/0.7.6/tutorial/printing.html#dot) .Should I add them back ?

File: doc/src/modules/printing.rst (hunk 1 above)
Comment (Shekharrajak) [inline doc/src/modules/printing.rst]: I found all the heading in capital in [printing docs](http://docs.sympy.org/0.7.6/modules/printing.html#module-sympy.printing.repr) yeah  I removed extra underlines.

File: doc/src/modules/printing.rst (hunk 3 above)
Comment (Shekharrajak) [inline doc/src/modules/printing.rst]: I followed this [examples code](https://github.com/sympy/sympy/blob/master/doc/src/modules/printing.rst#fortran-printing) ,when I was getting checks failed.

**Comment by moorepants** [comment]
Please fix the Travis failures.

**Comment by gxyd** [comment]
I am closing this PR, since PR's #10225 and #10253 address the issue.

**Pull request closed without merging**